src/operator/softmax_output-inl.h

```diff
@@ -43,6 +46,14 @@ struct SoftmaxOutputParam : public dmlc::Parameter<SoftmaxOutputParam> {
     DMLC_DECLARE_FIELD(use_ignore).set_default(false)
     .describe("If set to true, the ignore_label value will not contribute "
       "to the backward gradient");
+    DMLC_DECLARE_FIELD(normalization)
+    .add_enum("null", softmaxout_enum::kNull)
+    .add_enum("batch", softmaxout_enum::kBatch)
+    .add_enum("valid", softmaxout_enum::kValid)
```

Comment:
Stop sending me e-mails Sent from my Huawei Mobile-------- Original Message --------Subject: Re: [dmlc/mxnet] [Op] Add softmax norm option (mimic caffe) (#2231)From: Bing Xu To: dmlc/mxnet CC: Redneckgirl96 ,Mention In src/operator/softmax_output-inl.h:
> @@ -43,6 +46,14 @@ struct SoftmaxOutputParam : public dmlc::Parameter<SoftmaxOutputParam> {
>      DMLC_DECLARE_FIELD(use_ignore).set_default(false)
>      .describe("If set to true, the ignore_label value will not contribute "
>        "to the backward gradient");
> +    DMLC_DECLARE_FIELD(normalization)
> +    .add_enum("null", softmaxout_enum::kNull)
> +    .add_enum("batch", softmaxout_enum::kBatch)
> +    .add_enum("valid", softmaxout_enum::kValid)

@tornadomeet Can we use grad_scale to norm it?

—You are receiving this because you were mentioned.Reply to this email directly or view it on GitHub